requirements/default.txt

```diff
@@ -7,9 +7,9 @@ anyjson==0.3.1
 # sha256: SjoIXs8fzSc2VzU4_6EU8fQzGzu73Wk4Hm4XLEnJdQ8
 Babel==0.9.6
 
-# billiard: tags/v2.7.3.34
-# sha256: YKE9uzfN1UL1JUZXDT451i6erJQVPRDdfCT_poTAVy0
-https://github.com/celery/billiard/archive/f24a943cb4b6992717fa8d1db7e220d576377631.tar.gz#egg=billiard==2.7.3.34
```

Comment:
Instead of updating to a different github tag, can we switch this to something from pypi, like `billiard==3.3.0.9`? I'd like to eventually get rid of all the requirements that start with `http://`. Same goes for the rest of the requirements you updated (where possible).